### PR TITLE
Live calc columns and cleanup. Minor restructuring.

### PR DESCRIPTION
**Pull request by colinnashcinchy**
Please include a summary of the change and issue. Please also include relevant motivation and context. List any required dependencies for this change.

Fixes live calculated column documentation issue

## Type

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Release notes
- [x] Doc update

# Versions/Backporting

Please describe the applicable versions for the change.

**Doc version**: 5.7, 5.6 (must be manual graft)

# Testing 

Describe any testing, if applicable.
# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have done an initial review.
- [ ] I have completed an assessment of possible topic pages affected by the change.
- [x] My changes generate no new warnings
- [ ] (Optional) I have validated any automated changes.
- [ ] (If applicable) I have cherry picked changes and made a PR for other documentation versions

# Notes

### CODE REVIEW

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md [hunk 1]
```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
...
-### Choice
+**Example**: 
+ 
+A live "Label" column will update instantly if any referenced data changes, affecting all rows and tables.
 
-You can create a choice column (single or multi-select) in Cinchy. In this scenario, you specify all your choices (1 per newline) in the table design. A user will only be able to select from the options provided.
+#### Limitations
 
+- Live columns consume more system resources.
+- Using user-defined functions in live calculated columns can cause errors if they reference other live calculated columns. Only use inbuilt functions in live columns if they reference other live columns.
```
Comment (colinnashcinchy) [inline guides-for-using-cinchy/builder-guides/creating-tables/columns.md:263]: Here is the major highlight.

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md [hunk 2]
```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
...
+{% hint style="info" %}
+These columns are similar to computed columns in SQL databases.
+{% endhint %}
+### Live vs Cached Calculated Columns
 
-When creating a calculated column, you will notice the option to have it cached or not, located under Advanced Settings. This was an option introduced in version 4.0 of the platform.
+#### Choose Your Calculation Type
 
-![](<../../../.gitbook/assets/image (304).png>)
+When creating a calculated column, you have two types to choose from: **cached** and **live**. This feature is accessible via the Advanced Settings and was a part of the 4.0 version update.
```
Comment (balmasi) [inline guides-for-using-cinchy/builder-guides/creating-tables/columns.md:232]: Did we standardize on calling non-cached CCs "live"? 

We don't show that anywhere in the interface

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md [hunk 3]
```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
...
 
-A **cached calculated column** stores your data for fast retrieval and querying. Calculated columns are defaulted to cached. This is an actual column in your table that's based on a defined CQL formula. This column will recalculate when data is changed **in the same row.**
+#### Cached Calculated Columns
 
-In the below example, the **Label** column is a calculated column that connects various name columns together. You can see that _Connect your data_ appears in each label. If you then wanted to change the name in row one from _Connect your data_ to _Connect **all** your data_, only that specific row would recalculate automatically to update the label. To update any other row (within this table or another) with a calculated column that references that data, you would need to manually make changes to each to prompt a recalculation.
+- **What It Does**: Speeds up data retrieval.
+- **How It's Stored**: As an actual column based on a CQL formula.
+- **When It Updates**: Updates only if the data in the same row changes.
```
Comment (balmasi) [inline guides-for-using-cinchy/builder-guides/creating-tables/columns.md:238]: One gotcha is if it's relying on a column from another table and the value in the other table changes, this cached CC will not get updated. I think we should call this out.

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md [hunk 4]
```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
...
-In the above example, if the **Label** column was a live calculation, the formula would run every time you refresh the screen or query the data. If changes have been made (in this example, changing _"Connect your data"_ to _"Connect **all** your data"_), the calculated column will automatically reflect them, regardless of if it references another row or even another table.
+#### Live Calculated Columns
 
-While live calculated columns can be powerful this way, they also have their drawbacks, including possibly throwing errors and being resource intensive. Sometimes live calculated columns can even break a table (for example if you have a live calculated column linking to **another** live calculated column, you may run into errors).
+- **What It Does**: Provides real-time data.
+- **How It's Stored**: As a formula executed on-the-fly during read or query.
+- **When It Updates**: Refreshes automatically upon every query or screen refresh.
 
-### Choice
+**Example**:
```
Comment (balmasi) [inline guides-for-using-cinchy/builder-guides/creating-tables/columns.md:256]: Similar to above, maybe we can provide some more opinionated guidance:

**When to use:**
-  Your calculated column depends on a value from a linked table and you always need the latest value from the linked table.
- Your table does not contain a large number of records _(how small? Maybe QA would know)_

**Review by balmasi — COMMENTED** [review]
Question about "Live" calculated columns nomenclature

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md (hunk 3 above)
Comment (colinnashcinchy) [reply guides-for-using-cinchy/builder-guides/creating-tables/columns.md:238]: Adding additional information.

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md (hunk 2 above)
Comment (colinnashcinchy) [reply guides-for-using-cinchy/builder-guides/creating-tables/columns.md:232]: No idea. Who is the owner of the column types? Live calculated columns might be a little confusing, especially when using the comparitive.

File: guides-for-using-cinchy/builder-guides/creating-tables/columns.md (hunk 3 above)
Comment (colinnashcinchy) [reply guides-for-using-cinchy/builder-guides/creating-tables/columns.md:238]: Added on line 246

**Review by balmasi — APPROVED** [review]
I'm ok with "live calculated" columns if we explicitly define it as a non-cached calculated column in the doc

**Pull request merged**